gateway/1.10/modules/ROOT/nav.adoc

```diff
@@ -117,6 +117,7 @@
 *** xref:policies-outbound-oauth.adoc[Credential Injection OAuth 2.0]
 *** xref:policies-outbound-circuit-breaker.adoc[Circuit Breaker Policy]
 *** xref:policies-outbound-upstream-idle-timeout.adoc[Upstream Idle Timeout]
+*** xref:policies-included-xml-threat-protection.adoc[XML Threat Protection]
```

Comment:
Avoid hard-coding titles unless you're changing them from the H1 text.

Reply:
We hard code them because the policy titles have "Policy" in them and we remove it from the left nav.